pages/docs/concepts/asyncapi-document/payload-schema.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Payload Schema
+weight: 239
+---
+
+The payload schema defines a message's format, data types, and properties to ensure that the payload follows a specific structure and data format.
+
+In the AsyncAPI specification, you can use schema languages such as JSON or Avro Schema to create the payload schema. Enabling consumers to gain insights into the structure and data types of the payload.
+
+You should use Avro schemas to handle complex messages and structured data models. Avro schemas support complex types like arrays and records. You can create Avro schemas by writing the JSON representation or using tools like Avro4s UI to generate the schema from valid JSON.
```

Comment:
Why recommend Avro for complex schemas? Actually, JSON Schema is more suitable for complex structures. I would be careful of recommending one over the other because it depends on multiple factors.

Reply:
Based on this feedback, it leads me to think we need to add a section for "Define JSON schema". (Is this accurate @fmvilas/ @derberg?

Reply:
So basically there are multiple schemas, in document that @kakabisht writes, I recommend to focus on default `AsyncAPI Schema` (that is basically the same - superset - as JSON Schema but with few additional fields), then you can have separate documents (I think easier to find, easier to digest) for JSON Schema, Avro, Protobuf

Reply:
Yes, but I'd still avoid "recommending" one over the other. I know it's been changed but mentioning it again for clarity.

Reply:
yes, no recommendations, just documenting what is possible, and what are the limitations

Reply:
Thank you for the feedback.